sesman/config.c

```diff
@@ -553,12 +600,19 @@ config_dump(struct config_sesman *config)
 
     /* Security configuration */
     g_writeln("Security configuration:");
-    g_writeln("    AllowRootLogin:           %d", sc->allow_root);
-    g_writeln("    MaxLoginRetry:            %d", sc->login_retry);
-    g_writeln("    AlwaysGroupCheck:         %d", sc->ts_always_group_check);
-    g_writeln("    RestrictOutboundClipboard: %d", sc->restrict_outbound_clipboard);
-
-    g_printf( "    TSUsersGroup:             ");
+    g_writeln("    AllowRootLogin:                   %d", sc->allow_root);
+    g_writeln("    MaxLoginRetry:                    %d", sc->login_retry);
+    g_writeln("    AlwaysGroupCheck:                 %d", sc->ts_always_group_check);
+    g_writeln("    RestrictOutboundClipboard:        %d", sc->restrict_outbound_clipboard);
```

Comment:
Can use the existing `g_bitmask_to_str()` o display bitmap values here.

Reply:
Fixed to use it.